FirebasePerformance/Tests/Unit/FPRAppActivityTrackerTest.m

```diff
@@ -51,6 +51,8 @@ - (void)testInstanceCreation {
 /** Validates if an active trace is available when the app is active. */
 - (void)testActiveTrace {
   NSNotificationCenter *defaultCenter = [NSNotificationCenter defaultCenter];
+  [defaultCenter postNotificationName:UIWindowDidBecomeVisibleNotification
```

Comment:
The reason is that cocoapods test runs with a host app and `UIWindowDidBecomeVisibleNotification` will be called. Because spm tests do not have a host app, we have to manually post this notification.